### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by itsmdshahin** [vercel/community#3731] [Account Issue] [answered]
### Question

Only showing my account " This user account is blocked." what should I do ?

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Providing additional information about your account and the project you are deploying can help accelerate the account verification process.
